pkg/policy/rule.go

```diff
@@ -422,9 +422,6 @@ func mergeIngress(policyCtx PolicyContext, ctx *SearchContext, fromEndpoints api
 	hostWildcardL7 := make([]string, 0, 2)
```

Comment:
nit:

```suggestion
	hostWildcardL7 := make([]string0, 0, 1)
```

as we're not going to add the second item to the slice anymore in any case.

Reply:
Sorry, I don't see which comment specifically you mean. This is not related to my previous comment AFAICT.

Reply:
In any case it's just a nitpick and not worth a respin. Resolving.